gangplank/internal/ocp/return.go

```diff
@@ -41,66 +41,68 @@ func (r *Return) Run(ctx context.Context, ws *workSpec) error {
 	if r.Minio == nil {
 		return nil
 	}
-	baseBuildDir := filepath.Join(cosaSrvDir, "builds")
-	b, path, err := cosa.ReadBuild(baseBuildDir, "", cosa.BuilderArch())
-	if err != nil {
-		return err
-	}
```

Comment:
Rather than moving the logic under the `if` at line 49, you can compare `err` for the exported `ErrNoBuildFound`, which is more idiomatic.
```
	b, path, err := cosa.ReadBuild(baseBuildDir, "", cosa.BuilderArch())
	if err == cosa. ErrNoBuildsFound {
                 return nil
        } else if err != nil {
		return err
	}
```

Reply:
fixed in latest upload.. will do more testing with it in a bit

Reply:
> will do more testing with it in a bit

seems to be working in local testing 👍